source/clear-linux/guides/network/vnc.rst

```diff
@@ -408,7 +409,7 @@ SSH into your host and launch VNC
 
       #. Launch Putty.
       #. Under the :guilabel:`Category` section, select :guilabel:`Session`.
-         See Figure 1.
+         See figure 1.
```

Comment:
Is it our convention to refer to figures like this?

Reply:
Meaning the capitalization of "figure"? Yes, standardizing after what @mvincerx  does.

Reply:
Um, well, I capitalize references to figures. Example: See Figure 1. I followed those who preceded me, so I'm can't explain why!

Reply:
That's why I asked... Uppercase refs to figs or tables is the BKM going back thru time, lol. Also, lowercase refs to figs or tables is not the convention described in our [guide](https://clearlinux.org/documentation/clear-linux/reference/collaboration/structure-formatting#images). Just want to be consistent...